src/content/basics/giant-swarm-support/index.md

```diff
@@ -1,43 +1,45 @@
 +++
 title = "Giant Swarm Support"
 description = "An explanation of how our support service works."
-date = "2019-04-16"
+date = "2019-11-24"
 type = "page"
 weight = 20
 categories = ["basics"]
 +++
 
 # Giant Swarm Support
 
-Our support service is one of the key differentiators from our competitors. We have built a platform which lets our customer spin up as many clusters as they need. These clusters are then always in the same predictable state. This becomes especially valuable once you need to run multiple clusters for different teams, environments, configurations or in multiple locations.
+We built and run a platform which lets our customers spin up as many clusters as they need. We always keep these clusters in the same predictable state. This becomes especially valuable once you need to run multiple clusters for different teams, on different environments, with different configurations or in multiple locations.
 
-We are always at the side of our customers during their whole journey. For us, having a great platform is as important as having someone to contact when problems occur. The following is a brief description of our support process.
+We accompany our customers throughout their journey. Having someone to contact when problems occur is as important to us as gaving a great product. It is actually the obvious complement to our product. The following is a brief description of our support process.
 
 ## Customer Support
 
-Our customer support relies on close interactions via Slack, to get bi-directional feedback as quickly as possible. This is our first line of contact to customers, who can ask all kinds of questions, not only in relations to the platform itself, but many more around their whole Cloud Native journey.
+Our customer support relies on close interactions via Slack, to get bi-directional feedback as quickly as possible. This is our first line of contact to customers. This interaction channel is used also used to ask all kinds of questions. The conversations are not limited to the platform itself, but can go broader and include the whole Cloud Native journey.
 
-Our support shifts go from 08:00 to 18:00 on weekdays, but we also often answer questions well outside those times. We have a rotating shift for support across the team, which is why we keep support focused on those channels with clear internal handovers.
+Our support shifts go from 08:00 to 18:00 on weekdays, but we also often answer questions well outside those times. We rotate support shifts across the team, which is why focus on support channels with clear internal handovers.
```

Comment:
I think you forgot the we in "which is why focus on support channels with clear internal handovers."
WDYT?